syntax = "proto3";

package microservice;

service MicroService{
  // A simple RPC.
  //
  // Obtains the MessageResponse at a given position.
 rpc GetMicroServiceResponse(Message) returns (MessageResponse) {}

}

message Message{
 string message = 1;
}

message MessageResponse{
 string body = 1;
 bool received = 2;
}